joss_paper/paper.md

```diff
@@ -0,0 +1,121 @@
+This is possible thanks to the `Zarr` [@Zarr:2024] package which allows 
+streaming array-like data from a variety of storage mediums and [Kerchunk](https://github.com/fsspec/kerchunk), 
+which extends the capability of `Zarr` by allowing it to read `HDF5` files.
+Because it implements the standard `MDAnalysis` trajectory reader API,
+`Zarrtraj` can leverage `Zarr`'s ability to read a file in parallel to perform analysis 
+algorithms in parallel using the "split-apply-combine" paradigm. In addition to the `H5MD` format, 
+`Zarrtraj` can stream and write trajectories in the experimental `ZarrMD` 
+format, which ports the `H5MD` layout to the `Zarr` filetype.
+
+One imported, `Zarrtraj` allows passing trajectory URLs just like ordinary files:
+```python
+import zarrtraj
+import MDAnalysis as mda
+
+u = mda.Universe("sample_topology.top", "s3://sample-bucket-name/trajectory.h5md")
+```
+Initial benchmarks show that `Zarrtraj` can iterate
+through an AWS S3 cloud trajectory (load into memory one frame at a time)
+at roughly 1/2 or 1/3 the speed it can iterate through the same trajectory from disk and roughly 
+1/5 to 1/10 the speed it can iterate through the same trajectory on disk in XTC format \autoref{fig:benchmark}.
+However, it should be noted that this speed is influenced by network latency and that
+writing parallelized algorithms can offset this loss of speed.
+
+![Benchmarks performed on a machine with 2 Intel Xeon 2.00GHz CPUs, 32GB of RAM, and an SSD configured with RAID 0.\label{fig:benchmark}](benchmark.png)
+
+With `Zarrtraj`, we envision research groups making their data publicly available 
+via a cloud URL so that anyone can reuse their trajectories and reproduce their results.
+Large databases, like MDDB and MDverse, can expose a URL associated with each 
+trajectory in their databases so that users can make a query and immediately use the resulting
+trajectories to run an analysis on the hits that match their search. Groups seeking to 
+collect a large volume of trajectory data to train machine learning models can make use
+of our tool to efficiently and inexpensively obtain the data they need from these published 
+URLs.
+
+This work builds on the existing `MDAnalysis` `H5MDReader`
+[@H5MDReader:2021], and similarly uses `NumPy` [@NumPy:2020] as a common interface in-between `MDAnalysis`
+and the file storage medium. `Zarrtraj` was inspired and made possible by similar efforts in the 
+geosciences community to align data practices with FAIR principles [@PANGEO:2022].
+
+
+# Acknowledgements
+Thank you to Dr. Jenna Swarthout Goddard for supporting the GSoC program at MDAnalysis. 
+Thank you to Martin Durant, author of Kerchunk, for helping refine and merge features in his upstream codebase 
+necessary for this project. Finally, thank you to Google for supporting the Google Summer of 
+Code program (GSoC) which provided financial support for this project.
```

Comment:
Funding acknowledgement:
```suggestion
necessary for this project. Some of the work was funded through the Google Summer of Code program (GSoC) 2024.
```

Reply:
I also reached out on the gsoc mentors list to ask for the officially preferred language.

Reply:
Alternative suggested language (from someone on the mentors list who used it in https://iopscience.iop.org/article/10.1088/2632-2153/ad76f8 ):
```suggestion
necessary for this project. LW was a participant in the Google Summer of Code 2024 program.
```

Reply:
I have not heard much more so I would go with the last version.